### ISSUE TITLE
Preoccupation with unusual objects

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7286]
**Preferred term label:**
Preoccupation with unusual objects

**Synonyms**

**Definition (free text, please give PubMed ID)**
Having a greater interest in parts of a toy/object than in using the object as intended. For example, spinning the wheels on a toy car rather than playing with the car as intended.
Lord, C., Rutter, M. & Others. Social communication questionnaire (SCQ). Torrance, CA: WPS (2003)

**Parent term (use hpo.jax.org/app)**
Abnormal interests (issue #7274)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by bcoleman96** [comment]
Recommend making this a child term of "fixated interest with abnormal focus".

**Comment by pnrobinson** [comment]
Adding term as
Persistent preoccupation with parts of objects
HP:4000077
